gaia-switch.js

```diff
@@ -44,11 +44,31 @@ module.exports = component.register('gaia-switch', {
     this.disabled = this.getAttribute('disabled');
     this.checked = this.getAttribute('checked');
 
+    // put non-critical operations in post-load time frame
+    // to save loadtime.
```

Comment:
Sorry I'm still struggling to follow. I'd like to know *why* these actions are worth deferring. Do you mean:

`process everything that doesn't affect user interaction after the component is created`?